dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -47,6 +47,17 @@ HRESULT CFSDropTarget::_CopyItems(IShellFolder * pSFFrom, UINT cidl,
     /* Build a double null terminated list of C strings from source paths */
     for (UINT i = 0; i < cidl; i++)
     {
+        SFGAOF rfg = SFGAO_FILESYSANCESTOR | SFGAO_FILESYSTEM;
+        ret = pSFFrom->GetAttributesOf(1, apidl[i], &rfg);
+        if (FAILED(ret))
+            goto cleanup;
+
+        if (!(rfg & (SFGAO_FILESYSANCESTOR | SFGAO_FILESYSTEM)))
```

Comment:
Only check for SFGAO_FILESYSTEM unless you can show a real life case where SFGAO_FILESYSANCESTOR is a valid thing to check here?

Reply:
Deleted `SFGAO_FILESYSANCESTOR` flag.